README.md

```diff
@@ -1 +1,60 @@
-# laravel-settings
+## Laravel Settings
+Move your env variables to database to change them easily using laravel nova
+## Requirements
+- PHP >= 8.0
+- Laravel >= 8.0
+- Laravel nova license registered on project
+## Installation
+- Require package with composer:
+```bash
+composer require letsgoi/laravel-settings
+```
+
+- Add `Letsgoi\LaravelSettings\SettingsServiceProvider::class`to `app.php`
+````php
+    /*
+     * Package Service Providers...
+     */
+    ...
+    Letsgoi\LaravelSettings\SettingsServiceProvider::class,
+````
+
+- Publish configuration:
+```bash
+php artisan vendor:publish --provider="Letsgoi\LaravelSettings\SettingsServiceProvider" --tag="migrations"
+```
+
+- Add `SettingNovaResource::class` on `NovaServiceProvider.php`
+````php
+    protected function resources(): void
+    {
+        Nova::resources([
+            ...
+            SettingNovaResource::class,
+        ]);
+    }
+````
+
+## Usage
+To register a new variable on Settings:
```

Comment:
I miss some explanation about cache and how its works

Reply:
added